templates/shared_services/airlock_notifier/porter.yaml

```diff
@@ -43,7 +43,7 @@ parameters:
   - name: smtp_username
     env: SMTP_USERNAME
     type: string
-  - name: smtp_password
+  - name: smtpPassword
     env: SMTP_PASSWORD
     type: string
     sensitive: true
```

Comment:
This is the attribute causing the change.

Reply:
I don't remember why @SharonHart marked it as sensitive at the time, but now:
1. If not marked as sensitive, it will be saved in clear text into the porter database which I think isn't desired.
2. We can (should) use this property when generating the template sent to the API and not just count on the developer to mark the property as sensitive in the template_schema.json file.